src/operations/bulk_write.ts

```diff
@@ -48,14 +46,12 @@ export class BulkWriteOperation extends AbstractCallbackOperation<BulkWriteResul
         bulk.raw(operations[i]);
       }
     } catch (err) {
-      return callback(err);
+      return err;
```

Comment:
I think this is what's causing the CI to be so red. We were getting a lot of failures surrounding bulk write validation and not just throwing the error here would account for that.

You can get rid of the try-catch entirely and just run the loop since any errors thrown will automatically get converted to rejected promises